includes/editor-templates/responsive-top-bar.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace Elementor;
+
+use Elementor\Core\Responsive\Responsive;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+//$breakpoints = Responsive::get_breakpoints();
+$breakpoints = [
+	'mobile' => __( 'Mobile', 'elementor' ),
+	'tablet' => __( 'Tablet', 'elementor' ),
+	'desktop' => __( 'Desktop', 'elementor' ),
+];
```

Comment:
But it returns irrelevant values for my use case.
I left it there because I am aware the current solution isn't right for the next phases. and we need to address those needs, but for now I think it's ok...

Reply:
Will be improved once responsive settings API is ready